cmd/geoipupdate/main.go

```diff
@@ -39,7 +38,12 @@ func main() {
 	}
 
 	config, err := geoipupdate.NewConfig(
-		args.ConfigFile, defaultDatabaseDirectory, args.DatabaseDirectory, args.Verbose)
+		args.ConfigFile,
+		defaultDatabaseDirectory,
+		args.DatabaseDirectory,
+		args.Verbose,
+		geoipupdate.WithParallelism(args.Parallelism),
```

Comment:
It would probably be nice to make all the arguments for default values to be optional. I suspect people using the code as a library wouldn't be setting most of them.

This can be done in a follow-up story though.